dev/set_matrix.py

```diff
@@ -419,6 +419,7 @@ def expand_config(config):
                         run=run,
                         package=pip_release,
                         version=ver,
+                        supported=Version(ver) <= Version(max_ver),
```

Comment:
When we update a package version, this field flips to `True` and the cross-version-tests script recognizes it as a new item. We could use this field to allow failure versions that we don't support by checking this field.